vj4/ui/templates/domain_main.html

```diff
@@ -126,32 +126,31 @@ <h1 class="section__title">{{ _('Discussion') }}</h1>
   {% if domain_id == vj4.model.builtin.DOMAIN_ID_SYSTEM %}
     <div class="section side">
       <div class="section__header">
-        <h1 class="section__title">{{ _('Recommanded') }}</h1>
+        <h1 class="section__title">{{ _('Recommended') }}</h1>
       </div>
       <div class="section__body"><ul class="group-list">
         <li class="group-list__item">
           <h2 class="section__title">{{ _('Chinese') }}</h2>
           <ol class="chip-list">
-            <li class="chip-list__item"><a class="typo-a" href="https://www.xuetangx.com/courses/course-v1:TsinghuaX+30240184+sp/about" target="_blank">清华 MOOC (上)</a></li>
-            <li class="chip-list__item"><a class="typo-a" href="https://www.xuetangx.com/courses/course-v1:TsinghuaX+30240184_2X+sp/about" target="_blank">清华 MOOC (下)</a></li>
+            <li class="chip-list__item"><a class="typo-a" href="https://www.xuetangx.com/courses/course-v1:TsinghuaX+30240184+sp/about" target="_blank">清华数据结构 MOOC</a></li>
```

Comment:
But it occupies too much space for two links. A link to the series could solve both problems.